bookshelf/src/main/webapp/WEB-INF/web.xml

```diff
@@ -21,6 +21,9 @@
     -->
     <!-- [START filters] -->
     <filter-mapping>
+        <filter-name>DatastoreSessionFilter</filter-name>
```

Comment:
Maybe we can add a comment that we need this rather than the default (server-local?) session handling to support multiple instances.